packages/sdk-react/src/ui/FileViewer/FileViewer.tsx

```diff
@@ -156,120 +71,53 @@ const FileViewerComponent = ({
   mimetype,
   name,
   rightIcon,
-  Document,
-  Page,
   onReloadCallback,
-}: FileViewerProps & AsyncReturnType<typeof loadReactPDF>) => {
-  const [ref, { width }] = useMeasure<HTMLDivElement>();
-  const scrollbarWidth = useScrollbarWidth();
-  const [numPages, setNumPages] = useState<number>(0);
-  const [pageNumber, setPageNumber] = useState<number>(1);
-  const [scale, setScale] = useState<number>(1);
-
-  function onDocumentLoadSuccess({ numPages }: { numPages: number }) {
-    setNumPages(numPages);
-  }
-
-  const changePage = (offset: number) => {
-    setPageNumber((prevPageNumber) => prevPageNumber + offset);
-  };
+}: FileViewerProps) => {
+  const pdfRef = useRef<HTMLDivElement>(null);
 
-  const onPreviousPage = () => changePage(-1);
-  const onNextPage = () => changePage(1);
-  const onZoomIn = () => {
-    const newScale = scale <= SCALE_MAX ? scale + SCALE_STEP : scale;
-
-    setScale(Math.round(newScale * 10) / 10);
-  };
-  const onZoomOut = () => {
-    const newScale = scale >= SCALE_MIN ? scale - SCALE_STEP : scale;
-
-    setScale(Math.round(newScale * 10) / 10);
-  };
-
-  const isPdf = mimetype === 'application/pdf';
+  useEffect(() => {
+    PDFObject.embed(url, pdfRef.current, {
+      fallbackLink: true,
+      pdfOpenParams: {
+        // eslint-disable-next-line lingui/no-unlocalized-strings
```

Comment:
It would be better exclude such cases in [packages/eslint-plugin/src/config/lingui.js](https://github.com/team-monite/monite-sdk/blob/8f74872734694e209f4651266a544842dabef90e/packages/eslint-plugin/src/config/lingui.js#L26)

Reply:
Done